{{ cookiecutter.repo_name }}/tox.ini

```diff
@@ -1,12 +1,19 @@
 [tox]
 isolated_build = true
-envlist = py{38,39,310,311,312}-django{32,42}-psycopg2,py312-django42-psycopg3,report
+envlist =
+    py{38,39,310,311,312}-django32-psycopg2
+    py{38,39,310,311,312}-django42-psycopg2
+    py312-django42-psycopg3
+    py{310,311,312}-django50-psycopg2
+    py312-django42-psycopg3
```

Comment:
I believe you meant to put django 5.0 in this one. I will follow up with a commit to address this so that I can get the ball rolling on applying this to all of the libraries